PicIt/models/CameraModel.swift

```diff
@@ -0,0 +1,74 @@
+//
```

Comment:
Essentially identical to  [SwiftCamera](https://github.com/rorodriguez116/SwiftCamera), just moved it to its own file.